services/docextractor/docextractor_test.go

```diff
@@ -180,7 +180,7 @@ func (te *failingExtractor) Extract(filename string, r io.ReadSeeker) (string, e
 }
 
 func TestExtractWithExtraExtractors(t *testing.T) {
-	t.Run("overrite existing extractor", func(t *testing.T) {
+	t.Run("override existing extractor", func(t *testing.T) {
```

Comment:
?

sometimes people want `overwrite`, sometimes `override`, it depends on whether the effect is to stomp on content or change behavior.